File: Clase 5 - Estructuras Condicionales/if anidados.py
edad = 17
tieneLicencia = False
if edad >= 18:
  print("Eres mayor de edad")
  if tieneLicencia:
      print("Tienes licencia")
  else:
      print("No tienes licencia")
else:
    print("Eres menor de edad")
print("Fin del programa.")

# elif
nota = 81
if nota >= 80:
    print("Alumno promocionado")
elif nota >= 60:
    print("Alumno aprobado")
elif nota >= 40:
    print("Alumno aprobado con reserva")
else:
    print("Alumno no aprobado")

File: Clase 5 - Estructuras Condicionales/if anidados.js
// if anidados
const edad = 17
const tieneLicencia = false
if (edad >= 18) {
  console.log('Eres mayor de edad')
  if (tieneLicencia) {
    console.log('Tienes licencia')
  } else {
    console.log('No tienes licencia')
  }
} else { console.log('Eres menor de edad') }
console.log('Fin del programa.')

// else if
const nota = 30
if (nota >= 80) {
  console.log('Alumno promocionado')
} else if (nota >= 60 /* && nota < 80 */) {
  console.log('Alumno aprobado')
} else if (nota >= 40 /* && nota < 60 */) {
  console.log('Alumno aprobado con reserva')
} else {
  console.log('Alumno no aprobado')
}
